docs/data/data-grid/api-object/UseGridApiContext.js

```diff
@@ -27,6 +27,7 @@ export default function UseGridApiContext() {
     <Box sx={{ height: 400, width: '100%' }}>
       <DataGrid
         {...data}
+        loading={loading}
```

Comment:
I wonder. It feels like this adds mental overhead to each demo. We could move `loading` to be provided by `data`?